backend/server/src/main/java/com/issuetracker/util/JwtUtil.java

```diff
@@ -14,6 +14,7 @@
     private static final String JWT_ISSUER = "jwtIssuer";
     private static final String USER_LOGIN = "login";
     private static final String USER_NAME = "name";
+    private static final String USER_AVATAR_URL = "profileImageUrl";
```

Comment:
```suggestion
    private static final String USER_AVATAR_URL = "avatarUrl";
```

한번 프론트분들께 수정을 요청해봅시다.